src/amo/constants.js

```diff
@@ -1,13 +1,37 @@
+/* @flow */
 // Action types.
-export const SHOW_EDIT_REVIEW_FORM = 'SHOW_EDIT_REVIEW_FORM';
-export const SHOW_REPLY_TO_REVIEW_FORM = 'SHOW_REPLY_TO_REVIEW_FORM';
-export const FETCH_REVIEWS = 'FETCH_REVIEWS';
-export const HIDE_EDIT_REVIEW_FORM = 'HIDE_EDIT_REVIEW_FORM';
-export const HIDE_REPLY_TO_REVIEW_FORM = 'HIDE_REPLY_TO_REVIEW_FORM';
-export const SET_ADDON_REVIEWS = 'SET_ADDON_REVIEWS';
-export const SET_REVIEW = 'SET_REVIEW';
-export const SET_REVIEW_REPLY = 'SET_REVIEW_REPLY';
-export const SEND_REPLY_TO_REVIEW = 'SEND_REPLY_TO_REVIEW';
+export const SHOW_EDIT_REVIEW_FORM: 'SHOW_EDIT_REVIEW_FORM' =
+  'SHOW_EDIT_REVIEW_FORM';
+export const SHOW_REPLY_TO_REVIEW_FORM: 'SHOW_REPLY_TO_REVIEW_FORM' =
+  'SHOW_REPLY_TO_REVIEW_FORM';
+export const FETCH_REVIEWS: 'FETCH_REVIEWS' = 'FETCH_REVIEWS';
+export const HIDE_EDIT_REVIEW_FORM: 'HIDE_EDIT_REVIEW_FORM' =
+  'HIDE_EDIT_REVIEW_FORM';
+export const HIDE_REPLY_TO_REVIEW_FORM: 'HIDE_REPLY_TO_REVIEW_FORM' =
+  'HIDE_REPLY_TO_REVIEW_FORM';
+export const SET_ADDON_REVIEWS: 'SET_ADDON_REVIEWS' = 'SET_ADDON_REVIEWS';
+export const SET_REVIEW: 'SET_REVIEW' = 'SET_REVIEW';
+export const SET_REVIEW_REPLY: 'SET_REVIEW_REPLY' = 'SET_REVIEW_REPLY';
+export const SET_REVIEW_WAS_FLAGGED: 'SET_REVIEW_WAS_FLAGGED' =
+  'SET_REVIEW_WAS_FLAGGED';
+export const SEND_REPLY_TO_REVIEW: 'SEND_REPLY_TO_REVIEW' =
+  'SEND_REPLY_TO_REVIEW';
+export const SEND_REVIEW_FLAG: 'SEND_REVIEW_FLAG' = 'SEND_REVIEW_FLAG';
+
+export const REVIEW_FLAG_REASON_SPAM: 'review_flag_reason_spam' =
```

Comment:
Any reason these are lowercase?

Reply:
They map to specific values that the API accepts https://addons-server.readthedocs.io/en/latest/topics/api/reviews.html#post--api-v3-reviews-review-(int-id)-flag-

Reply:
Ah, right on. Might be worth a comment explaining what these are then, but makes sense.